### PR TITLE
'Reconnect to Sync' text field acts as a button #487

### PR DESCRIPTION
**Pull request by utkarsh858**
fixes : #487 

The text field will work as same as the Sync Button.

### CODE REVIEW

**Comment by utkarsh858** [comment]
@vladikoff  Can you look into it .

File: src/sidebar/menu.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/**
+#/**
```
Comment (Natim) [inline src/sidebar/menu.css]: I believe this should not be present here.

File: src/sidebar/panel.js [hunk 2]
```diff
@@ -69,11 +69,16 @@ ClassicEditor.create(document.querySelector('#editor'), {
         }
         ignoreNextLoadEvent = false;
       });
-
+      
+      savingIndicator.onclick = () => {
+        enableSyncAction(editor);
```
Comment (Natim) [inline src/sidebar/panel.js]: I believe this should be deactivated during saving (as for enableSync)

Comment (Natim) [reply src/sidebar/panel.js]: Nevermind it seems to be handled in the enableSyncAction itself: https://github.com/utkarsh858/notes/blob/6135f7841191671facbfcefd12f91d5f033c9e46/src/sidebar/panel.js#L238

File: src/sidebar/panel.js [hunk 3]
```diff
@@ -205,6 +210,7 @@ function reconnectSync () {
   isAuthenticated = false;
   setAnimation(false, true, true); // animateSyncIcon, syncingLayout, warning
   savingIndicator.textContent = browser.i18n.getMessage('reconnectSync');
+  savingIndicator.style="cursor:pointer;";
```
Comment (Natim) [inline src/sidebar/panel.js]: If it works for all sidebar actions, why put the cursor only for reconnect to sync?

Comment (utkarsh858) [reply src/sidebar/panel.js]: Because the task for that text field only.

**Pull request merged**